### PR TITLE
Add *_PUBLIC_URI environment variable

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

**Comment by gip** [comment]
@justmoon should review this

Associated PRs will be needed on each Interledger component to update the docs I think?

File: test/configSpec.js [hunk 1]
```diff
@@ -160,6 +160,31 @@ describe('Config', () => {
...
+    })
+
+    it('PUBLIC_URI - no port or path', () => {
+      process.env.UNIT_TEST_OVERRIDE = 'true'
+      process.env.PUBLIC_URI = 'http://www.example.com'
+      const _config = Config.loadConfig()
+      expect(_config.getIn(['server', 'public_secure'])).to.equal(false)
+      expect(_config.getIn(['server', 'public_host']))
+        .to.equal('www.example.com')
+      expect(_config.getIn(['server', 'public_port'])).to.equal(80)
```
Comment (justmoon) [inline test/configSpec.js]: Might be worth adding a test for a https URI to make sure it defaults to port 443 there. But definitely a nice-to-have. :)

**Comment by justmoon** [comment]
Awesome! That's going to make our users' job a *lot* easier.

LGTM

File: test/configSpec.js (hunk 1 above)
Comment (clark800) [inline test/configSpec.js]: Added a test case

**Pull request merged**